--- a/lib/cloudevent/generate.js
+++ b/lib/cloudevent/generate.js
@@ -48,7 +48,7 @@
         delete cloudEventPartial.data;
     }
     return cloudEventPartial
-        ? (0, ts_deepmerge_1.default)(generatedCloudEvent, cloudEventPartial)
+        ? (0, ts_deepmerge_1.merge)(generatedCloudEvent, cloudEventPartial)
         : generatedCloudEvent;
 }
 function shouldDeleteUserSuppliedData(generatedCloudEvent, cloudEventPartial) {
--- a/package.json
+++ b/package.json
@@ -40,7 +40,7 @@
   "dependencies": {
     "@types/lodash": "^4.14.104",
     "lodash": "^4.17.5",
-    "ts-deepmerge": "^2.0.1"
+    "ts-deepmerge": "8.0.0"
   },
   "devDependencies": {
     "@types/chai": "~4.2.4",
